### PR TITLE
Add support for postgres11

### PR DESCRIPTION
**Pull request by easel**
Test against AWS Aurora Postgres v 11.7

### CODE REVIEW

File: scripts/install_pg_repack.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-wget -q -O pg_repack.zip "https://api.pgxn.org/dist/pg_repack/1.4.3/pg_repack-1.4.3.zip"
+wget -q -O pg_repack.zip "https://api.pgxn.org/dist/pg_repack/1.4.4/pg_repack-1.4.4.zip"
 unzip pg_repack.zip && rm pg_repack.zip
 cd pg_repack-*
 make && make install
 cd ..
 rm -rf pg_repack-*
-apt-get remove --auto-remove -y make unzip gcc libssl-dev zlib1g-dev
\ No newline at end of file
+apt-get remove --auto-remove -y make unzip gcc libssl-dev zlib1g-dev
```
Comment (hartmut-co-uk) [inline scripts/install_pg_repack.sh:10]: 👍

**Review by hartmut-co-uk — CHANGES_REQUESTED** [review]
Many thanks, any particular reason not to use latest, which is 1.4.6?
https://github.com/reorg/pg_repack/releases

If possible - could you test if 1.4.6 also works with pg11 / your setup?

**Comment by easel** [comment]
I can definitely test it. The reason I built 1.4.4 was that's the version AWS is using with their postgres11 instances. In an ideal scenario we'd tag 1.4.1 with postgres 10, 1.4.4 with postgres 11 and 1.4.5 / 1.4.6 with their corresponding releases as well.

**Comment by hartmut-co-uk** [comment]
Agree, good point. As I understand 1.4.6 should be fully backwards compatible though.
Please let me know if 1.4.6 works with pg11 if possible.

Also let me check if I've had integration with docker hub - or also if I might add Github actions + Github package registry support. 🤔

**Comment by hartmut-co-uk** [comment]
right - I've had it linked with docker hub: https://hub.docker.com/repository/docker/hartmutcouk/pg-repack-docker
so should work with tagging..
I'll try to get this sorted on the WE if I can find time.

Many thanks for the contribution! 👍

**Comment by easel** [comment]
I'm pretty sure pg_repack itself is compatible, but the issue is you won't always have control of the extension version loaded into the db (with AWS RDS for example). 

My assumption about this container was that it was a convenient way to isolate the "client" from the server for such scenarios, so ideally we'd have a version built for every version of pg_repack along with the most appropriate postgres version.

**Review by hartmut-co-uk — APPROVED** [review]

**Pull request merged**